.rubocop_todo.yml

```diff
@@ -56,6 +56,13 @@ Layout/SpaceInsideParens:
   Exclude:
     - 'test/loader_test.rb'
 
+# Offense count: 3
+Lint/MissingSuper:
+  Exclude:
+    - 'test/executor_test.rb'
+    - 'test/loader_test.rb'
+    - 'test/support/loaders.rb'
```

Comment:
Although not calling `super` isn't recommended, we do need to keep supporting it for backwards compatibility, so I avoided updating these offenses.